uint8 X_LEFT = 0
uint8 X_RIGHT = 1
uint8 Y_FRONT = 2
uint8 Y_REAR = 3
uint8 Z_FRONT_LEFT = 4
uint8 Z_FRONT_RIGHT = 5
uint8 Z_REAR_LEFT = 6
uint8 Z_REAR_RIGHT = 7
uint8 COUNT = 8
